serializations.md

```diff
@@ -732,6 +753,28 @@ tree.
 
 ```
 
+###### Channel off-chain pinned block for on-chain environment
+
+Each update can be optionally based on an on-chain block that is used for an
+environment for on-chain lookups: accounts, oracles queries, responses and
+etc. This is to improve the process of both reaching an off-chain consensus
+regarding changes and having more predictable on-chain forced progress
```

Comment:
* improve the process of ... reaching an off-chain consensus regarding changes

Ok.

* [improve the process of] having more predictable on-chain forced progress executions

Not clear to me - as I understand for force progress the block hash in the force progress tx - not the included payload - is used.

Reply:
I will rework it